### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by DanielElisenberg**
# UPDATE DOCS
Updated user docs in accordance with updates to the code. Bumped major version to release the changes to PyPI. Will bump the patch version when s380 have made updates to the centrally defined unit type variables and temporal attribute variables.

### 4.0.0 => 5.0.0 Changelog
* Unit types are centrally defined
* Identifier is always defined by only one field "unitType"
* Measure variable can either be defined as before, or with "unitType", "name" and "description",  where "name" and "description" overwrites the central definitions of these fields.
* shortName removed from root fields and measure variable fields. Now injected from filename and validated to only contain upper case A-Z, number 0-9 or underscores
* CodeList contains the list directly rather than a nested "codeListItems"-list
* Subjectfields are simplified from having "shortName", "name", "description" to having just one multilanguage field. The reason for this is that we only ever used the "name" field.
* SensitivityLevel is added as a required root level field and must contain on of PERSON_GENERAL, PERSON_SPECIAL, PUBLIC or NONPUBLIC as requested by SIKT
* The centrally defined unit type variable contain a new field "pseudonymized" that, like the variables themselves, are centrally defined
* Updates to documentation and examples that reflect the changes in the component

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,52 +1,39 @@
...
-* **name**: Human readable name(Label) of the attribute column. Used as the column label in the ROSE client. Example: "Startdato".
-* **description**: Description of the column contents.
-* **dataType**: DataType for the values in the column. One of: ["STRING", "LONG", "DOUBLE", "DATE"]
-* **format**: More detailed description of the values. For example a regular expression.
-* **uriDefinition**: Link to external resource describing the measure.
-* **unitType**: See definition below.
-* **valueDomain**: See definition below.
+
+### MEASURE VARIABLES (with unitType)
+You might find that some of your datasets contain a unitType in the measure column as well. Let's say you have a dataset PERSON_MOR where the identifier-column is a population of unitType "PERSON", and the measure column is a population of unitType "PERSON". The measure here is representing the populations mothers. Then you may define it as such:
```
Comment (pawbu) [inline docs/README.md:33]: identifier column

File: docs/README.md [hunk 2]
```diff
@@ -99,12 +86,16 @@ The second example belongs to the measure variable of a dataset where the measur
...
 
 
-### UNIT TYPE
-Description of the unit the data describes. A "fødselsnummer" can be used to identify a PERSON. Another example would be an "organisasjonsnummer" that would be used to identify a "FORETAK".
-* **shortName**: Machine readable name.
-* **name**: Human readable name.
-* **description**: Description of the unit type.
-
+### UNIT TYPES
+* **PERSON**: Representation of a person in the microdata platform. Columns with this unit type should contain FNR.
```
Comment (pawbu) [inline docs/README.md:90]: Maybe translate all types to English in the descriptions? 🤔

File: docs/USAGE.md [hunk 3]
```diff
@@ -119,102 +119,48 @@ This will only check if all required fields are present, and that the metadata f
 
 
 ## Use metadata references
+```
```
Comment (pawbu) [inline docs/USAGE.md:122]: 💯

**Review by pawbu — APPROVED** [review]
Looks good, some minor comments.

Maybe refer to the project as microdata.no (platform) instead of just microdata (platform).

File: docs/README.md (hunk 2 above)
Comment (DanielElisenberg) [reply docs/README.md]: We can attempt this in the future once we get the unitTypes with languagefields for both Norwegian and English 😅 I have no idea what to translate FORETAK/VIRKSOMHET to according to the definition used in the platform f.ex.

**Pull request merged**